pkgs/by-name/tt/tt/package.nix

```
@@ -0,0 +1,28 @@
+{ lib, fetchFromGitHub, buildGoModule }:
+
+buildGoModule rec{
```

Minor blank-space nit:
```suggestion
buildGoModule rec {
```